src/glmtools.jl

```diff
@@ -79,20 +79,24 @@ mutable struct NegativeBinomialLink  <: Link
     θ::Float64
 end
 
+import Base.==
+==(lnk1::NegativeBinomialLink, lnk2::NegativeBinomialLink) =
```

Comment:
Can't you just make `NegativeBinomialLink` immutable instead?

Also if you keep it:
```suggestion
Base.:(==)(lnk1::NegativeBinomialLink, lnk2::NegativeBinomialLink) =
```

Reply:
Sure.